src/index.js

```diff
@@ -74,10 +77,14 @@ export default function ({ types: t, traverse: _traverse }) {
         const { parentPath } = path;
         const explicitReturn = parentPath.isReturnStatement();
         const implicitReturn = parentPath.isArrowFunctionExpression();
-
-        const openingElement = elementOpenCall(path.get("openingElement"), this);
-        const closingElement = elementCloseCall(path.get("openingElement"), this);
-        const children = buildChildren(path.get("children"), this);
+        const openingElementPath = path.get("openingElement");
+
+        const openingElement = elementOpenCall(openingElementPath, this);
+        const closingElement = elementCloseCall(openingElementPath, this);
+        const isSkipping = hasSkip(openingElementPath.get("attributes"), this) ||
+          isComponent(openingElementPath.get("name"), this);
```

Comment:
Interesting, does this mean that whenever a component tag is found, its contents will be skipped? I actually use the contents on my side, so this would cause a lot of trouble for me :( Would it be possible to safe guard this change behind an option?